charts/partials/templates/_proxy.tpl

```diff
@@ -73,8 +73,10 @@ env:
 - name: LINKERD2_PROXY_IDENTITY_DIR
   value: /var/run/linkerd/identity/end-entity
 - name: LINKERD2_PROXY_IDENTITY_TRUST_ANCHORS
-  value: |
-  {{- required "Please provide the identity trust anchors" .Values.identityTrustAnchorsPEM | trim | nindent 4 }}
+  valueFrom:
+    configMapKeyRef:
+      name: linkerd-identity-trust-roots
+      key: ca.pem
```

Comment:
Won't this change how injected proxies are configured? Or is this only used with the core control plane? This configmap won't be accessible for non-controller proxies.